src/interfaces/correction-filing-interface.ts

```diff
@@ -21,6 +21,7 @@ export interface CorrectionFilingIF {
     correctedFilingId: string
     correctedFilingType: string
     correctedFilingDate: string
+    type?: string // ? for firms only
```

Comment:
This should be an enum.

Reply:
Is this for firms only, or is type (client/staff) prompted for all corrections now, too?